learn-more.css

```diff
@@ -1,91 +0,0 @@
-#header h1 {
-  text-align: center;
-  font-size: xx-large;
-}
-
-#header h4 {
-  text-align: center;
-
-}
-
-#main-left {
-  justify-content: left;
-  
-}
-
-#main-right {
-  justify-content: right;
-  
-}
-
-#main-right img {
-  width: 15vw;
-  height: 35vh;
-}
-
-#main-left img {
-  width: 15vw;
-  height: 35vh;
-}
-
-
-
-/*------------------------footer----------------------------- */
```

Comment:
Great to see this big page break comments. These make navigation within the program so much nicer.